percona-xtradb-cluster-8.0/dockerdir/etc/mysql/node.cnf

```diff
@@ -1,5 +1,7 @@
 [mysqld]
 
+#admin-address=noname
```

Comment:
we should add this line similar to `wsrep_node_incoming_address` https://github.com/percona/percona-xtradb-cluster-operator/blob/master/build/pxc-configure-pxc.sh#L66-L67

reason - old releases should work with the new operator